column/api/controller/run_controller.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import uuid
+
+from flask import request
+from flask_restful import abort
+from flask_restful import Resource
+
+from column.api import backend
+from column.api import manager
+
+LOG = logging.getLogger(__name__)
+
+
+class Run(Resource):
+    """Column api run resource class
+
+    Attributes:
+        backend_store (obj): backend interface for retrieving run info
+    """
+
+    def __init__(self):
+        self.backend_store = backend.get_store()
+
+    def get(self, id):
+        """Get run by id"""
+        run = self.backend_store.get_run(id)
+        return run if run else abort(404,
+            message="Run {} doesn't exist".format(id))
+
+
+class RunList(Resource):
+    """Column api runlist resource class
+
+    Attributes:
+        backend_store (obj): backend interface for retrieving run info
+        manager (obj): manager interface for calling column library method
+                       to run playbook
+    """
+
+    def __init__(self):
+        self.backend_store = backend.get_store()
+        self.manager = manager.get_manager('run')
+
+    def get(self):
+        """Get run list"""
+        LOG.info('Returning all ansible runs')
+        return self.backend_store.list_runs()
+
+    def post(self):
+        """Trigger a new run"""
+        # TODO(sihan) Add Run factory for validating the payload fields
```

Comment:
Will this be done in a future patch?  Also user id sihan doesn't exist in github